testing/run

```diff
@@ -79,8 +79,8 @@ prechecks() {
     (git describe HEAD --long --always || echo "git missing") > ${ARTIFACT_DIR}/ci_artifact.git_version
 
     if [[ "${PULL_NUMBER:-}" ]]; then
-        curl -sSf "https://api.github.com/repos/openshift-psap/ci-artifacts/pulls/$PULL_NUMBER" -o "${ARTIFACT_DIR}/pull_request.json"
-        curl -sSf "https://api.github.com/repos/openshift-psap/ci-artifacts/issues/420/comments" -o "${ARTIFACT_DIR}/pull_request-comments.json"
+        curl -sSf "https://api.github.com/repos/openshift-psap/ci-artifacts/pulls/$PULL_NUMBER" -o "${ARTIFACT_DIR}/pull_request.json" || true
```

Comment:
Isn't this introducing silent failures?  Shouldn't at least the failure be logged somewhere?

Reply:
Actually, I take this back (`-S`).  So fine with me. :)

Reply:
good point, but the failure isn't "silent", `curl -sSf` will log an error message in `stderr`, as at the bottom of [this log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/30732/rehearse-30732-periodic-ci-openshift-psap-ci-artifacts-master-ods-cleanup-velero-buckets/1550478314798321664/build-log.txt)
```
curl: (22) The requested URL returned error: 404 
```
true, that's not the clearest message (I thought `set -x` was active in this file, but it's not)

Reply:
@fcami agrees with you offline,

I'll change this to:
```
curl ... || echo "Failed to download ..."
```